<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtHelpConfigUI</class>
 <widget class="QWidget" name="QtHelpConfigUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>443</height>
   </rect>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <property name="margin" stdset="0">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="boxAutoLoad">
     <property name="title">
      <string comment="@title:group">Load Qt Help Documentation from System Locations</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelLoadQtApiDocs">
        <property name="text">
         <string comment="@option:check">&amp;Load Qt API documentation:</string>
        </property>
        <property name="buddy">
         <cstring>loadQtDocsCheckBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="loadQtDocsCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelLoadFromDirectory">
        <property name="text">
         <string comment="@label:textbox">Load Qt Help files from directory:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KUrlRequester" name="qchSearchDir"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="KMessageWidget" name="messageAvailabilityQtDocs">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="boxQchManage">
     <property name="title">
      <string comment="@title:group">Install Additional Documentation Files</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="qchTable">
            <property name="editTriggers">
             <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="columnCount">
             <number>5</number>
            </property>
            <property name="showGrid" stdset="0">
             <bool>false</bool>
            </property>
            <attribute name="headerVisible">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string comment="@title:column">Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string comment="@title:column">Path</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">IconName</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">Ghns</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true"/>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="tableCtrlLayout">
            <item>
             <widget class="QPushButton" name="addButton">
              <property name="text">
               <string comment="@action:button">Add</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="topMargin">
             <number>0</number>
            </property>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
